plugin.yaml

```diff
@@ -20,7 +20,15 @@ dsl_definitions:
       type: boolean
       default: true
 
+# Will patch_id property be added to the ClientConfig?
 data_types:
+  cloudify.types.starlingx.ClientConfigDetails:
+    derived_from: cloudify.types.starlingx.ClientConfig
+    properties:
+      patch_id:
+        type: string
+        description:
+        required: true
```

Comment:
Why is this called a `client_config`? I suggest that this will be `cloudify.types.starlingx.Patch`. Also data_types do not have `derived_from` in schema. See for example `cloudify.types.starlingx.WRCP`.

Reply:
Ok, I added that because I assumed that this will be some sort of inheritance. Removed and changed to   `cloudify.types.starlingx.Patch`.